packages/gatsby-recipes/src/providers/npm/package-json.js

```diff
@@ -58,18 +59,20 @@ exports.schema = schema
 exports.validate = validate
 
 module.exports.plan = async ({ root }, { id, name, value }) => {
+  // let parsedValue = value
```

Comment:
Not sure if this comment is meant to still be in.

Reply:
nope! Thanks!